apps/sms/index.html

```diff
@@ -63,7 +63,7 @@
     <link rel="stylesheet" type="text/css" href="shared/elements/gaia_sim_picker/style.css">
     -->
     <!-- Shared code -->
-    <script defer src="shared/js/l10n.js"></script>
+    <script type="application/javascript;version=1.8" defer src="shared/js/l20n.js"></script>
```

Comment:
are you sure it's needed nowadays ?
I'm quite sure it's not needed for `const`; not sure for `let` though.
or maybe gaia applications are special-cased ?